app/components/framework/FrameworkOutput/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { Error, getErrorObject, analyzeErrors } from '@togglecorp/toggle-form';
+import { ListView } from '@the-deep/deep-ui';
+
+import NonFieldError from '#components/NonFieldError';
+import { Widget } from '#types/newAnalyticalFramework';
+import WidgetPreview from '#components/framework/WidgetPreview';
+
+import styles from './styles.css';
+
+interface WidgetProps {
+    className?: string;
+    widget: Widget;
+    clientId: string;
+    onWidgetValueChange: (value: unknown, widgetName: string) => void;
```

Comment:
Why is this type unknown? This should directly be extended from WidgetPreview instead.